examples/ex1.cpp

```diff
@@ -144,7 +144,16 @@ int main(int argc, char *argv[])
    bool delete_fec;
    if (order > 0)
    {
-      fec = new H1_FECollection(order, dim);
+      if (mesh.NURBSext)
+      {
+         fec = new NURBSFECollection(order);
```

Comment:
Should this change be made also in `ex1p` and `ex11p`?

Reply:
To prevent compatiblity/acceptance issues,  I have changed `ex1.cpp` back. And modified the FEspace initialization of some of the `miniapp/nurbs` to showcase the required changes.

Changes to the examples can be a separate low impact PR, with perhaps a difficult discussion ...